hometask3.py

```diff
@@ -0,0 +1,34 @@
+"""Hometask format and f-strings."""
+import logging
+
+logging.basicConfig(level=logging.INFO, format='[%(asctime)s] %(levelname)s: %(message)s')
+
+# we have norway text in old style formatting
+# re-write the same text as:
+# #1 string with format() call
+# #2 f-string
+# use linter(https://github.com/wemake-services/wemake-python-styleguide)
+# to check your new created python module for possible linter errors
+# try to run code from pycharm/command line
+
+norway_text = 'Automatisering akselererer %syeblikket da roboter vil erobre planeten v%sr. (%s)' % ('ø', 'å', 'Æ')
```

Comment:
Here and below linter gives us smth aka `E501 line too long (114 > 79 characters)` - you need to fix it